packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -118,26 +133,134 @@ fillDate(
 
 >- `selectCheckbox()` - Enzyme helper that fires a change event with a value for a checkbox at the given name
 
+## Example unit tests
+
+VSP provides helper utilities to make writing tests easier. The following example unit tests illustrate some of these helpers. You can find these unit tests at `src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx`:
+
+```js
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import { DefinitionTester, fillData, selectRadio, fillDate } from '../../../../platform/testing/unit/schemaform-utils.jsx';
+import formConfig from '../config/form.js';
+
+describe('VIC veteran information', () => {
+  const { schema, uiSchema } = formConfig.chapters.veteranInformation.pages.veteranInformation;
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data=\{{}}
```

Comment:
I could be wrong, but I suspect that `\` character is not supposed to be there. 

```
- data=\{{}}
+ data={{}}
```

Reply:
Removed